### PR TITLE
Inline editorial corrections, promte substantive to Proposed Corrections

### PR DESCRIPTION
**Pull request by svgeesus**
As discussed on today's call.

### CODE REVIEW

File: woff2/index.html [hunk 1]
```diff
@@ -1188,14 +1183,14 @@ <h4 id="CompositeGlyph0">Decoding of Composite Glyphs</h4>
       the coordinate values are stored as part of the glyph data stream using a
       variable-length encoding format consuming a total of 2-5 bytes per point.</p>
 
-    <div class="correction" id="c4">
-      <span class="marker">Candidate Correction 4:</span>
+    <div class="proposed correction" id="p2">
+      <span class="marker">Proposed Correction 2:</span>
```
Comment (vlevantovsky) [inline woff2/index.html:1187]: Should the class itself be identified as "correction"?

**Review by vlevantovsky — APPROVED** [review]
Overall, all changes look good. Once we resolve the differences in class definitions (currently the are defined differently - "correction" for p1 and "proposed correction" for p2), we can fix and merge the changes.

**Comment by svgeesus** [comment]
> Overall, all changes look good. Once we resolve the differences in class definitions (currently the are defined differently - "correction" for p1 and "proposed correction" for p2), we can fix and merge the changes.

Good catch, fixed

**Comment by vlevantovsky** [comment]
Can you please double check the latest changes? C5 is editorial and should not be even marked as proposed correction. 
My earlier comment was related to changes in original "c2" -> "p1" and "c4" -> "p2". In "p1", the class definition remained "correction" while the marker was edited to "Proposed correction" (which is correct, IMO), while in "p2" - both the class definition and the class marker were changed to "proposed correction".

**Pull request merged**